libminifi/src/utils/BackTrace.cpp

```diff
@@ -54,8 +54,10 @@ namespace {
 }  // namespace
 #endif
 
+#ifndef HAS_EXECINFO
+void pull_trace(uint8_t) {
+#else  // HAS_EXECINFO
 void pull_trace(uint8_t frames_to_skip /* = 1 */) {
```

Comment:
I think we can avoid these ifndef-s and (void) workarounds with the new [C++ attribute: maybe_unused (since C++17)](https://en.cppreference.com/w/cpp/language/attributes/maybe_unused)
```suggestion
void pull_trace([[maybe_unused]] uint8_t frames_to_skip /* = 1 */) {
```

Reply:
Thanks for the idea! I didn't realize `[[maybe_unused]]` could be used on parameters, not just return values. Fixed now.